js/interface.js

```diff
@@ -0,0 +1,90 @@
+// Get the modal
+let modal = document.getElementById("myModal");
+
+let openModal0 = document.getElementById("m0");
+let openModal1 = document.getElementById("m1");
+let openModal2 = document.getElementById("m2");
+let openModal3 = document.getElementById("m3");
+let openModal4 = document.getElementById("m4");
+let openModal5 = document.getElementById("m5");
+let openModal6 = document.getElementById("m6");
+let openModal7 = document.getElementById("m7");
+let openModal8 = document.getElementById("m8");
+let openModal9 = document.getElementById("m9");
```

Comment:
Is there a way to DRY this code?
Actually, this applies to the whole file.

You could have a loop that:
* creates the element
* sets a click handler to the element
* adds it on the page.